src/lib/components/buy_product/BuyProduct.js

```diff
@@ -41,14 +44,18 @@ export default class BuyProduct extends Component {
   }
 
   render() {
-    const { className, design } = this.props;
+    this.productHelper = new ProductHelper();
```

Comment:
This should move in the constructor if we don't need to create  ProductHelper every time on re-render